travis/ipa-init.sh

```diff
@@ -22,7 +22,12 @@
 set -e
 
 # Enable IPA COPR repo
-dnf copr enable -y @freeipa/freeipa-4-7
+# NOTE: IPA does not build for F30
+. /etc/os-release
+if [[ `echo "$VERSION_ID"` -ge 31 ]]
```

Comment:
So should this line be `30` then instead of `31`?

Reply:
I just worked with IPA to enable it for F30. I will be removing this condition :-)

To answer your question, `-ge` stands for greater than or equals. So, F31 is right :-)

Reply:
Now it is right, but when we first discussed this, your comment said `F30` didn't work but your check was saying `F30` worked... ;-)